src/main/cms/cms_menu_osd.c

```diff
@@ -84,6 +84,9 @@ OSD_Entry menuOsdActiveElemsEntries[] =
     {"GPS LON",            OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_GPS_LON], 0},
     {"HOME DIR",           OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_HOME_DIR], 0},
     {"HOME DIST",          OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_HOME_DIST], 0},
+#ifdef USE_SERIALRX_CRSF
```

Comment:
Why is this inside the `USE_GPS` condition?

Reply:
the link quality is messured in %. 
300 -200 = 150hz 
100-0 = 50 hz